### PR TITLE
Active admin

### PR DESCRIPTION
**Pull request by lexelby**
This sets up active-admin on /admin.  By default it shows incidents, but you can also view categories.  We can add the ability to edit other objects like Remediations, Retrospectives, etc.  I also added a `Sync` link -- not sure if this works 100% yet.

No authentication on this yet!  That'll come in a later PR.

### CODE REVIEW

File: app/admin/incident.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ActiveAdmin.register Incident do
+  menu priority: 1
+
+  # Don't allow creating incidents, since syncing should be the only way of
+  # creating incidents in Retrodot.
+  actions :all, except: [:new]
+
+  member_action :sync, method: :post do
+    Mediators::Incident::Syncher.run(incident: resource[:incident_id])
+    redirect_to resource_path, notice: "Synced!"
```
Comment (joshuatobin) [inline app/admin/incident.rb]: Nice!

File: app/support/mediators/incident/syncher.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ def call
       update_incident
 
     rescue Excon::Error
-      Rails.logging.error($!, fn: "call", at: "update_incident", incident_id: @options[:incident])
+      Rails.logger.error(exception: $!, fn: "call", at: "update_incident", incident_id: @options[:incident])
       raise $!
```
Comment (joshuatobin) [inline app/support/mediators/incident/syncher.rb]: 👍 I fixed this in another PR somewhere. nice catch

File: config/initializers/active_admin.rb [hunk 3]
```diff
@@ -0,0 +1,272 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
+  #
+  # Set the title that is displayed on the main layout
+  # for each of the active admin pages.
+  #
+  config.site_title = "Retrodot"
```
Comment (joshuatobin) [inline config/initializers/active_admin.rb]: Should we configertor-ize some of these?

**Comment by joshuatobin** [comment]
This is awesome 👍

File: Gemfile [hunk 4]
```diff
@@ -18,6 +18,8 @@ gem "haml-rails", "~> 0.9"
 gem "excon", "0.50.1"
 gem "configerator"
 gem "business_time"
+gem "activeadmin", "~> 1.0.0.pre4"
+gem 'inherited_resources', github: 'activeadmin/inherited_resources'
```
Comment (reidmix) [inline Gemfile]: TIL

File: app/admin/incident.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  # add a "sync" button to the "view incident" page
+  action_item :sync, only: :show do
+    link_to 'Sync', sync_admin_incident_path(resource)
+  end
```
Comment (reidmix) [inline app/admin/incident.rb]: Aha!

File: config/initializers/active_admin.rb (hunk 3 above)
Comment (reidmix) [inline config/initializers/active_admin.rb]: Yup.

**Comment by reidmix** [comment]
This is awesome!

File: app/assets/javascripts/active_admin.js.coffee [hunk 6]
```diff
@@ -0,0 +1 @@
+#= require active_admin/base
```
Comment (reidmix) [inline app/assets/javascripts/active_admin.js.coffee]: Anyway to keep coffeescript out?

File: Gemfile (hunk 4 above)
Comment (lexelby) [inline Gemfile]: Me too.  I got that from the activeadmin docs.

File: app/assets/javascripts/active_admin.js.coffee (hunk 6 above)
Comment (lexelby) [inline app/assets/javascripts/active_admin.js.coffee]: TBH, I have no idea.  I'll look into it.

File: app/support/mediators/incident/syncher.rb (hunk 2 above)
Comment (lexelby) [inline app/support/mediators/incident/syncher.rb]: Excellent.  At some point in the merging, these two will cancel each other out :)

File: app/assets/javascripts/active_admin.js.coffee (hunk 6 above)
Comment (lexelby) [inline app/assets/javascripts/active_admin.js.coffee]: Apparently we can! Just deleting the file doesn't seem to break anything. ;)

I think we'd only need to have the file if we did some weird custom stuff, which we're not gonna do.

**Comment by lexelby** [comment]
Okay, I think that addresses everything you folks commented on.

File: app/assets/javascripts/active_admin.js.coffee (hunk 6 above)
Comment (reidmix) [inline app/assets/javascripts/active_admin.js.coffee]: And I think you can just use `js` to do the manifest and Rails will do the right thing.   You can see this in status.  This is part of the Asset pipeline (which I need to read up on to refamiliarize myself).

**Pull request merged**